### PR TITLE
use new Aliyun DNS query API

### PR DESCRIPTION
**Pull request by loganhz**
we need to use another query API now as Aliyun DNS API is returning a different format now
[issue](https://github.com/denverdino/aliyungo/issues/148)

### CODE REVIEW

**Comment by loganhz** [comment]
@yasker Can you help to review this PR? it's a fix for Aliyun DNS.
Aliyun changed the response format recently.

File: providers/alidns/alidns.go [hunk 1]
```diff
@@ -111,8 +112,11 @@ func (a *AlidnsProvider) GetRecords() ([]utils.DnsRecord, error) {
 			fqdn = fmt.Sprintf("%s.%s.", rec.RR, a.rootDomainName)
 		}
 
-		recordTTLs[fqdn] = map[string]int{}
-		recordTTLs[fqdn][rec.Type] = int(rec.TTL)
+    recordTTLs[fqdn] = map[string]int{}
```
Comment (yasker) [inline providers/alidns/alidns.go]: Format issue?

**Comment by yasker** [comment]
@logan 

Please provide a build for @Tracy1994 to test.

**Comment by loganhz** [comment]
@yasker my bad.
I really thought I did `go fmt` stuff. 
The build is ready for test. I will talk to Tracy on Monday.
thanks

**Comment by Tracy1994** [comment]
Bug  https://github.com/rancher/rancher/issues/9797 had already fixed.
This PR has been tested and passed.

**Pull request merged**